crates/nu-command/src/strings/size.rs

```diff
@@ -165,7 +165,11 @@ fn size(
         move |v| match v.as_string() {
             Ok(s) => counter(&s, span),
             Err(_) => Value::Error {
-                error: ShellError::PipelineMismatch("string".into(), span, span),
+                error: ShellError::PipelineMismatch(
+                    "string".into(),
+                    span,
+                    v.span().unwrap_or_else(|_| Span::new(0, 0)),
```

Comment:
please don't use `Span::new(0, 0)` as it will point into some unknown source code.

Instead, just default to `span`.

Reply:
Sorry. There was already a bunch of other uses of that elsewhere, so I assumed it was OK.
![image](https://user-images.githubusercontent.com/83939/203470784-86e6ade6-774e-4aa9-9e76-ddf4c5fa902d.png)

Reply:
Yeah, we should remove those 😅